mmv1/products/firebase/api.yaml

```diff
@@ -152,3 +152,66 @@ objects:
           Immutable. The globally unique, Firebase-assigned identifier of the App.
 
           This identifier should be treated as an opaque token, as the data format is not specified.
+  - !ruby/object:Api::Resource
+    name: 'AppleApp'
+    min_version: beta
+    base_url: projects/{{project}}/iosApps
+    self_link: '{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A Google Cloud Firebase Apple application instance
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/'
```

Comment:
probably use https://firebase.google.com/docs/ios/setup instead (or both) as it walks through the Apple ecosystem parts which are more directly applicable then the general Firebase docs landing page.